logging-k8s-stdout-otlp-json/k3d.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+./build.sh
+k3d cluster create jsonlogging || k3d cluster start jsonlogging
+k3d image import -c jsonlogging dice:1.1-SNAPSHOT
```

Comment:
since this file already has a dependency on the dice image, does it make sense to just inline the one line here?